### PR TITLE
Add db migration step to docker-entrypoint.sh

### PR DESCRIPTION
**Pull request by 00Chaotic**
Previously, when running through docker, if the database did not exist, it would be created using `db:setup`. However, once the db does exist, there is no provision for running migrations, which leads to an error unless the migrations are run, or the containers are torn down and built up again (which would run `db:setup` again, thus implementing the new migrations anyway).

This change adds a migration step to the `docker-entrypoint.sh` file so that `rails db:migrate` is run if the database exists.

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -1,11 +1,18 @@
 #!/bin/sh
 set -e
 
+if [ -f /usr/src/app/tmp/pids/server.pid ]; then
+  rm /usr/src/app/tmp/pids/server.pid
+fi
+
```
Comment (cassar) [inline docker-entrypoint.sh:7]: Is this change related to this issue?

Comment (00Chaotic) [reply docker-entrypoint.sh:7]: No, this was intended to be part of https://github.com/govhackaustralia/hackerspace3/pull/361 but doesn't seem to have been included.

From my understanding, when the service (or container?) isn't shut down properly, an `A server is already running.` error is displayed when you try starting it again, causing it to instantly exit. Since there are no downsides to this approach and it would probably take longer to find out and address _why_ the error occurs in the first place, I've just decided to go with this solution.
